Update community code of conduct (#359)
obsidianmd/obsidian-docs: en/Obsidian/Community code of conduct.md

These are the behaviors we do not tolerate.

### Serious offenses

> Once verified, these offenses will result in an immediate ban from our community.

Reviewer: I'd cut the "Once verified, " here and below. I think reasonable people would assume due diligence across the implementation of the entire CoC.

> Once verified, these offenses will result in an immediate ban from our community.

Author: hmm not necessarily a channel, could be the forum as well

> Once verified, these offenses will result in an immediate ban from our community.

Reviewer 2: I think this means "communication channels," not "discord channels"

> Once verified, these offenses will result in an immediate ban from our community.

Author: ok, in that case I think we also need to be clear about what the channels are, so that the reader knows the scope.

The channels I have in mind are:
- Discord
- Forum (English and Chinese)
- Reddit

> Once verified, these offenses will result in an immediate ban from our community.

Reviewer: The conversation below between @marcusolsson and @ericaxu is also relevant here:
> > Just a nitpick word of caution when referring to "our community". While a company can act as leaders or stewards within a community, I don't believe any company can own a community.
> 
> I meant it more like in the sense of "our friend", in which case clearly our friend is not owned by us. Is the suggestion to swap out "our community" with "Obsidian's community" or "the community" or "the Obsidian community"?

"from the official Obsidian community" would also cover it without us having to keep track of what exactly what we call "official." I don't think it's worth our trouble to try to keep an up-to-date itemized list in the docs.

> Once verified, these offenses will result in an immediate ban from our community.

Author: Why remove "once verified"?

See comment from Marcus here: https://github.com/obsidianmd/obsidian-docs/pull/359#discussion_r1062653392

- **Harassment**: including offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, or religion, as well as sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual or romantic attention.
- **Threats**: threatening someone physically or verbally. For example, threatening to publicize sensitive information about someone’s personal life.

### Other offenses

> Once verified, these offenses will result in a warning. On second offence, an immediate ban will be issued.

Reviewer: Rewrite for active voice, Canadian spelling of "offences", and to offer more flexibility on enforcement:

> Users who commit these offences will be sent a warning. Repeated offences will result in the user being banned from the community.

- **Blatant -isms**: saying things that are explicitly racist, sexist, homophobic, etc. For example, arguing that some people are less intelligent because of their gender, race or religion. Small mistakes made in conversation are not code of conduct violations. However, repeating something after it has been pointed out to you, or antagonizing or arguing with someone who has pointed out your subtle -ism is considered a minor offense.
- **Belligerence**: deliberately attempting to make others feel bad, name-calling, singling out others for derision or exclusion. For example, telling someone they’re not a real programmer or that they don’t belong.
- **Being especially unpleasant**: for example, if we’ve received reports from multiple members of annoying or rude behavior. ^432ea1

### Other behaviors

> We strongly discourage these behaviors. Please don’t do them. These behaviors do not result in a ban or warning, but they will greatly increase one’s chance of [[#^432ea1|being especially unpleasant]].

Reviewer: I like this a lot!

> We strongly discourage these behaviors. Please don’t do them. These behaviors do not result in a ban or warning, but they will greatly increase one’s chance of [[#^432ea1|being especially unpleasant]].

Author: "strongly discourage you from engaging in them"?

> We strongly discourage these behaviors. Please don’t do them. These behaviors do not result in a ban or warning, but they will greatly increase one’s chance of [[#^432ea1|being especially unpleasant]].

Author: @marcusolsson did you see my comment?

> We strongly discourage these behaviors. Please don’t do them. These behaviors do not result in a ban or warning, but they will greatly increase one’s chance of [[#^432ea1|being especially unpleasant]].

Reviewer 3: LGTM!

- **"Feed me"**: asking others to solve problems for you without putting in any effort yourself. This is particularly egregious when the problem-solving is non-trivial and yet the user begs for someone else to do the work for them.
- **Campaigning**: continuously making the same requests. This applies to both feature requests and to requests for help. Do not post the same question in multiple places in short timeframes, and do not ask developers for the same features or bug fixes repeatedly.
- **Parasitism**: self-promoting of yours or others' initiatives without contributing to the community otherwise. We encourage self-promotion (it is awesome to see what Obsidian users are up to) but will not tolerate users whose only purpose in the community is to post links or solicitations to initiatives outside the community.

## Encouraged behaviors

Over the years of running our community, we’ve noticed some behaviors that positively contribute to the community.

If you’re looking to improve your communication skills as well as becoming a better community citizen, you can:

> - **Try before asking**: if you’re asking for help, please first try searching the help docs and Discord/forum history to see if the question has always been asked. When asking the question, sharing what you have tried but doesn’t work would help others help you.

Reviewer 3: ```suggestion
- **Try before asking**: before asking for help, please first try searching the help docs and Discord/forum history to see if the question has already been asked. Help others help you, by sharing what you've already tried.
```

- **Keep an open mind**: because of how customizable Obsidian can be, some people might have a completely different set-up or workflow from you. Do not judge others for their choices.

> - **Step away when needed**: the community is a place for us to grow together, not a debate competitions. There are no arguments that you have to win. If a discussion turns into an argument and escalates, try your best to step away and be the bigger person.

Reviewer 3: ```suggestion
- **Step away when needed**: the community is a place for us to grow together, not a debate competition. There are no arguments that you have to win. If a discussion escalates into an argument, try your best to step away and be the bigger person.
```

> There are no arguments that you have to win.

I'm not sure I understand this line. Do you mean something like "arguments don't have winners or losers"?

Author: Good point, I'll remove it to avoid confusion

> - **Step away when needed**: the community is a place for us to grow together, not a debate competitions. There are no arguments that you have to win. If a discussion turns into an argument and escalates, try your best to step away and be the bigger person.

Reviewer: > [...] try your best to step away. 

Cut the "be the bigger person" part as it suggests a superiority/piousness competition.

> - **Step away when needed**: the community is a place for us to grow together, not a debate competitions. There are no arguments that you have to win. If a discussion turns into an argument and escalates, try your best to step away and be the bigger person.

Author: I feel like this is giving too much advice, will leave it short but take the "heated" word suggestion. Thanks!

- **Threats**: threatening someone physically or verbally. For example, threatening to publicize sensitive information about someone’s personal life.

### Other offenses

Once verified, the following offenses will result in a warning. On second offense, an immediate ban will be issued.

- **Blatant -isms**: saying things that are explicitly racist, sexist, homophobic, etc. For example, arguing that some people are less intelligent because of their gender, race or religion. Small mistakes made in conversation are not code of conduct violations. However, repeating something after it has been pointed out to you, or antagonizing or arguing with someone who has pointed out your subtle -ism is considered a minor offense.
- **Belligerence**: deliberately attempting to make others feel bad, name-calling, singling out others for derision or exclusion. For example, telling someone they’re not a real programmer or that they don’t belong.
- **Being especially unpleasant**: for example, if we’ve received reports from multiple members of annoying or rude behavior. ^432ea1

> - **Asking for attention from the team for non-emergencies**: this includes pinging or messaging developers and moderators for questions, bug reports, and feature requests. As much as we want to help everyone, we have a sizable community, and we want to reserve our attention for the most critical matters. You can ping or direct message us for things like:
> 	- Plugin security related issues
> 	- To report violations of the community code of conduct

Reviewer: ```suggestion
- **Asking for the attention of members of the Obsidian team**: this includes unsolicited tagging, "pinging", or direct messaging developers and moderators for questions, bug reports, and feature requests. As much as we want to help everyone, we have a sizeable community, and the team's attention must be reserved for critical matters. You can ping or direct message members of the team for things like security-related issues or to report violations of the community code of conduct. Otherwise, please post about the issue in an appropriate channel on the forum or the Discord server.
```

> A verified major offense results in an immediate ban.
> A verified minor offense results in a warning; after that a second verified violation results in a ban.

> If our moderator team witnesses or receives a report about [[#serious offenses]], we will contact the offender to have a conversation with them and verify what has happened, and they will be removed from our community.

Reviewer 3: > [...] have a conversation with them and verify what has happened, and they will be removed from our community

This potentially sounds like they will be banned regardless of their defense?

Author: I'll change it too "we will contact the offender to have a conversation with them and verify what has happened. If verified, they will be removed from our community".

Major offenses are only harassments and threats. I don't think there's much room for defense as they are pretty clear cut.
